test/parallel/test-stringbytes-external.js

```diff
@@ -1,4 +1,5 @@
 'use strict';
+// Flags: --expose_internals
```

Comment:
No idea, it was too long ago. Don't bother reviewing, I'm gonna redo this from scratch